src/routes/login.tsx

```diff
@@ -1,6 +1,13 @@
 import Authenticated from '@/components/authenticated'
 
-const goBack = () => window.history.back()
+const useQuery = () => {
+  return new URLSearchParams(window.location.search);
```

Comment:
In this project we've been using [`qss`](https://github.com/lukeed/qss#qssdecodequery) to encode/decode query strings. It might make sense to switch to the built-in `URLSearchParams` eventually, but for now can we continue to use `qss` for consistency?